phpcs-ruleset.xml

```diff
@@ -2,4 +2,10 @@
 <ruleset name="Behat for WordPress coding standards">
   <rule ref="PSR1" />
   <rule ref="PSR2" />
+  <rule ref="PSR2.Methods.FunctionCallSignature.SpaceAfterCloseBracket">
```

Comment:
Looks great but I don't understand what this does -- isn't SpaceAfterCloseBracket not included as part of the general PSR2 rule reference? You mention a severity, is that something we should set somewhere?